code/datums/components/jetpack.dm

```diff
@@ -167,7 +167,7 @@
 	if (get_dir(source, backup) == movement_dir || source.loc == backup.loc)
 		return
 
-	if (!source.client.intended_direction || (source.client.intended_direction & get_dir(source, backup)))
+	if (!source.client?.intended_direction || (source.client.intended_direction & get_dir(source, backup)))
```

Comment:
Does the ? on the first check return and skip the other check?

Reply:
yes